the-collab-lab/tcl-38-smart-shopping-list
# implement copy token to clipboard with button in nav component.

## Description

This PR adds a button to the nav (so it appears on both Item List and Add Item pages when a user has a token) that allows the user to copy their token to the clipboard.

We referred to this article for the clipboard functions: https://blog.logrocket.com/implementing-copy-to-clipboard-in-react-with-clipboard-api/

## Related Issue
Closes #38 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates


| before | after |
|---|---|
|![Screen Shot 2022-02-28 at 1 44 52 PM](https://user-images.githubusercontent.com/83732773/156063920-85052ecc-3797-4fd6-afce-4883a5793c00.png) | <img width="316" alt="Screen Shot 2022-02-28 at 1 45 20 PM" src="https://user-images.githubusercontent.com/83732773/156063990-2fb9952f-5b89-4d62-b6bf-8e17fef21ffe.png"> |


## Testing Steps / QA Criteria

Pull down branch. And either create token or make sure you have a token already in local storage. Click the Share Token button. You should see an alert that says your token has been copied to the clipboard. Paste your token somewhere to see it.
